src/galaxysrv.hpp

```diff
@@ -45,10 +46,11 @@
 
 #include "platform.hpp"
 
-// gal.peers.peers_add();
+#include "netbuf.hpp"
+
 
 /// @owner magaNet_user
-class c_galaxysrv : public c_galaxysrv_peers, c_galaxysrv_cables, c_galaxysrv_p2p, c_galaxysrv_e2e {
```

Comment:
NACK, for now decided on that design
(can re-evaluate later after this all new engine works)